### PR TITLE
Trees list & adding new trees

### LINKED ISSUE #21: Add mobile form for creating a tree

### PR DESCRIPTION
**Pull request by Dragemil**
Closes #21 

Screenshot_20201208_224922

### CODE REVIEW

**Comment by BartoszChrostowski** [comment]
Some tests failed :<

File: mobile/lib/contracts/tree/contracts.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+@JsonSerializable()
+class TreeItemDTO {
+  final String id;
+  final String photoUri;
+  final String treeName;
+
+  TreeItemDTO({this.id, this.photoUri, this.treeName});
+
+  factory TreeItemDTO.fromJson(Map<String, dynamic> json) =>
```
Comment (BartoszChrostowski) [inline mobile/lib/contracts/tree/contracts.dart:14]: Seems cumbersome, are u sure there's no other way?

**Review by BartoszChrostowski — APPROVED** [review]

**Comment by BartoszChrostowski** [comment]
Please fix tests

File: mobile/lib/contracts/tree/contracts.dart (hunk 1 above)
Comment (Dragemil) [reply mobile/lib/contracts/tree/contracts.dart:14]: I don't know about what exactly your talking about, but there isn't any way to generate contracts

**Pull request merged**